tests/unit-tests/stripe/tests-give-stripe-admin-helpers.php

```diff
@@ -30,32 +30,6 @@ public function setUp() {
 		$this->gateways = give_get_option( 'gateways', array() );
 	}
 
-	/**
-	 * Unit test for function give_stripe_is_connected();
-	 *
-	 * @since 2.5.0
-	 *
-	 * @return void
-	 */
-	public function test_give_stripe_is_connected() {
```

Comment:
Any reason for removing this test?

Reply:
@bradp Because we no longer use this fn and is deprecated